cmake/modules/hunter_finalize.cmake

```diff
@@ -86,6 +86,10 @@ macro(hunter_finalize)
     set(_pkg_config_dir2 "${HUNTER_INSTALL_PREFIX}/share/pkgconfig")
     # This info is also in hunter_autotools_project.cmake
     set(ENV{PKG_CONFIG_LIBDIR} "${_pkg_config_dir1}:${_pkg_config_dir2}")
+
+    # Automatically generated Find sripts from pkg-config modules
+    # go into ${HUNTER_INSTALL_PREFIX}/lib/cmake, so:
+    list(APPEND CMAKE_MODULE_PATH "${HUNTER_INSTALL_PREFIX}/lib/cmake")
```

Comment:
Let's use `${HUNTER_INSTALL_PREFIX}/cmake/Hunter` here.

Just for your info:
* http://cgold.readthedocs.io/en/latest/tutorials/install/layout.html